src/components/addSourceWizard/hardcodedComponents/oci/costManagement.js

```diff
@@ -203,10 +203,10 @@ export const PopulateBucket = () => {
           {intl.formatMessage({
             id: 'cost.oci.populateBucket.subtitle1',
             defaultMessage:
-              'In your Oracle Cloud account, create a VM and run a script similar to the one from this github repository:',
+              'In your Oracle Cloud account, create a VM and run a script provided in the documentation:',
```

Comment:
here, the linter wants just put "defaultMessage" and the string on the same line